cmd/autoowners/main_test.go

```diff
@@ -1,11 +1,13 @@
 package main
 
 import (
+	"fmt"
 	"reflect"
 	"testing"
 
 	"k8s.io/apimachinery/pkg/util/diff"
 	"k8s.io/apimachinery/pkg/util/sets"
+	"k8s.io/test-infra/prow/github"
```

Comment:
nit: split the test-infra and apimachinery libraries. Add a line between them.

Reply:
Aren't the rules?

* build-in pkgs
* pkgs from site_1
* pkgs from site_n
* local pkgs

So `apimachinery` and `test-infra` are from `k8s.io` and no need for the line in between?

Reply:
We tend to split also the repositories, not only the org. Not so important though.

Reply:
Maybe I do not know the rules at all ... :)

Why do we have a line between `apimachinery` and `client-go` but not `api` and `apimachinery`?

https://github.com/openshift/ci-tools/blob/b5e3576894e5b3aab0d8609e38024a5d1977d160/cmd/ci-operator/main.go#L26-L32

Reply:
in some part of the code, we have it and in another part, we don't. :)

Reply:
I think sometimes this is a matter of personal preference and that is not harmful. The only solid rule we have is to separate standard library, 3rd party libs and local imports, but sometimes it makes sense to separate a part of 3rd party libs to its separate block, especially when the 3rd party import block is long.

Reply:
Make sense.
I will keep those in mind from now on.